### PR TITLE
Introduction to Android App Development with Basic4Android B4A Part1

### PR DESCRIPTION
**Pull request by Tsanguu**
added index.md and image files

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by Tsanguu** [comment]
This fixes #757

File: articles/Introduction-to-android-app-development-basic4android-b4a-part1/index.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+End Sub
+```
+
+In the function above, we check if the textboxes are empty. If empty, we prompt the user to enter some numbers. When the above condition is met, we perform an arithmetic operation on the numbers and display the results in a label.
+
+**Output**
+
+![B4A App Screenshot](/engineering-education/introduction-to-android-app-development-basic4android-b4a-part1/b4a-sample-app-output.jpg)
+
+You can find the complete source code on [Github](). You can also find more example projects in the [B4A community](https://www.b4x.com/android/forum/threads/b4a-projects-source-code.29614/).
```
Comment (kayere) [inline articles/Introduction-to-android-app-development-basic4android-b4a-part1/index.md:181]: Add the Github Link.

**Review by kayere — COMMENTED** [review]
@Tsanguu - Great article, easy to follow through. I have learned a lot about B4A.
I have made some changes to the article.  Please go through them; I hope it will help improve the quality of your next articles. I'd suggest you use the image linking format I have used (in your future articles). 
Otherwise, add the GitHub link and your article will be good to go.

**Comment by Tsanguu** [comment]
@kayere On the image linking it is well noted. I have also made the suggested change.

**Pull request merged**